site/src/components/DeploySettingsLayout/Sidebar.tsx

```diff
@@ -93,6 +94,14 @@ export const Sidebar: React.FC = () => {
       >
         Security
       </SidebarNavItem>
+      {dashboard.experiments.includes("deployment_health_page") && (
+        <SidebarNavItem
+          href="/health"
+          icon={<SidebarNavItemIcon icon={MonitorHeartOutlined} />}
+        >
+          Health checks
```

Comment:
FYI When you are talking about monitoring systems, the more common form is "healthchecks". You may want to adjust other places too.

In this case, I would just name it accordingly to the page, so Health.